package.json

```diff
@@ -54,6 +54,7 @@
     "eslint-plugin-react-hooks": "^2.5.0",
     "lerna": "^3.20.2",
     "prettier": "1.19.1",
+    "raw-loader": "^4.0.1",
```

Comment:
This also does not seem to be used anywhere.

Reply:
removed